tests/active_features/connection:info.feature

```diff
@@ -0,0 +1,64 @@
+@environment @connection
+Feature: Environment Connection Info Command
+  In order to see connection parameters for remotely accessing the environment of a given site
+  As a Terminus user
+  I want a command to view the current connection parameters.
+
+  Background: I am authenticated and have a site named [[test_site_name]]
+
+  @vcr site_connection-info
+  Scenario: Show all connection info for a site's environment
+    Given I am authenticated
+    And a site named: [[test_site_name]]
+    When I run "terminus connection:info [[test_site_name]].[[test_env_name]]"
+    Then I should see a table with the headers: Parameter, Connection Info
+    And I should see a table with rows like:
```

Comment:
No 😞 . I was headed that way since the output is fairly predictable and we could have strict assertions on the table but this one just asserts that the value exists in the table but does not assert its position or even if the output row count matches the expectation. That's why I added the "rows like" rather than "rows equal" 😄 . I figured a good source of inspiration might be found in the unit tests around the RowsOfFields formatter but that exploration was left as a TODO.